internal/services/newrelic/new_relic_monitor_resource.go

```diff
@@ -105,11 +105,11 @@ func (r NewRelicMonitorResource) Arguments() map[string]*pluginsdk.Schema {
 						Type:     pluginsdk.TypeString,
 						Optional: true,
 						ForceNew: true,
-						Default:  string(monitors.BillingCycleMONTHLY),
+						Default:  "MONTHLY",
 						ValidateFunc: validation.StringInSlice([]string{
-							string(monitors.BillingCycleMONTHLY),
-							string(monitors.BillingCycleWEEKLY),
-							string(monitors.BillingCycleYEARLY),
+							"MONTHLY",
+							"WEEKLY",
+							"YEARLY",
```

Comment:
is there a reason these are no longer constants? could we open an issue on the rest specs asking and link it here ?

Reply:
Hi @katbyte , from service team, earlier there was limit on values that this field can have, now that limitation has lifted. There will be new values in the future.

Reply:
I'm not sure why that means there is no longer an enum/defined in the rest API specs? i cannot see any reason why these values shouldn't be captures there. could we place open an issue on the rest specs + link it here?